exercise_08/pipeline.txt

```diff
@@ -0,0 +1,25 @@
+import groovy.json.JsonSlurper
+
+pipeline {
+    agent {label 'jenkins-slave-archita'}
+
+    stages {
+        stage('Fetch') {
+            steps {
+                script {
+                    try {
+                        URL api_url_obj = new URL("https://jsonplaceholder.typicode.com/todos")
+                        def data= new JsonSlurper().parse(api_url_obj)
+                        data.each { obj ->
```

Comment:
naming could still be better

data could be called todos
obj could be called todo